### ISSUE TITLE
Woocommerce order is empty

### ISSUE THREAD

**Issue by nrgoutfitters** [woocommerce/woocommerce#17012]
Hi there!

We are testing our eshop and found out that the order screen is empty. We did a POS purchase and once the purchase is done, the order screen look empty. We have updated our wordpress and theme to the latest version.

Cheers,

**Comment by reigelgallarde** [comment]
Copy and paste the system status report from **WooCommerce > System Status** in WordPress admin here.

**Comment by nrgoutfitters** [comment]
`
### WordPress Environment ###

Home URL: http://nrgoutfitters.com
Site URL: http://nrgoutfitters.com
WC Version: 3.1.2
Log Directory Writable: ✔
WP Version: 4.8.2
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_US

### Server Environment ###

Server Info: Apache
PHP Version: ❌ 5.4.32 - We recommend a minimum PHP version of 5.6. See: How
to update your PHP version
PHP Post Max Size: 8 MB
PHP Time Limit: 60
PHP Max Input Vars: 1000
cURL Version: 7.19.7
NSS/3.13.1.0

SUHOSIN Installed: –
MySQL Version: ❌ 5.5.39 - We recommend a minimum MySQL version of 5.6. See:
WordPress requirements
Max Upload Size: 8 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ❌ Your server does not have the SoapClient class enabled - some
gateway plugins which use SOAP may not work as expected.
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 3.1.2
WC Database Prefix: wp_
woocommerce_sessions: ✔
woocommerce_api_keys: ✔
woocommerce_attribute_taxonomies: ✔
woocommerce_downloadable_product_permissions: ✔
woocommerce_order_items: ✔
woocommerce_order_itemmeta: ✔
woocommerce_tax_rates: ✔
woocommerce_tax_rate_locations: ✔
woocommerce_shipping_zones: ✔
woocommerce_shipping_zone_locations: ✔
woocommerce_shipping_zone_methods: ✔
woocommerce_payment_tokens: ✔
woocommerce_payment_tokenmeta: ✔
MaxMind GeoIP Database: ❌ The MaxMind GeoIP Database does not exist -
Geolocation will not function. You can download and install it manually
from http://dev.maxmind.com/geoip/legacy/geolite/ to the path: . Scroll
down to "Downloads" and download the "Binary / gzip" file next to "GeoLite
Country". Please remember to uncompress GeoIP.dat.gz and upload the
GeoIP.dat file only.

### Security ###

Secure connection (HTTPS): ❌Your store is not using HTTPS. Learn more about
HTTPS and SSL Certificates.
Hide errors from visitors: ✔

### Active Plugins (8) ###

Polylang: by Frédéric Demarle – 2.2.3
Contact Form 7: by Takayuki Miyoshi – 4.9
Huge IT Google Map: by Huge-IT – 2.2.6
Kirki Toolkit: by Aristeides Stathopoulos – 3.0.10
WooCommerce POS: by kilbot – 0.4.14
WooCommerce Price Based on Country (Basic): by Oscar Gare – 1.6.19
WooCommerce: by Automattic – 3.1.2
WordPress Carousel: by ShapedPlugin – 1.4.5

### Settings ###

API Enabled: ✔
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)


### WC Pages ###

Shop base: ❌ Page visibility should be public
Cart: #6 - /cart/
Checkout: #7 - /checkout/
My account: #8 - /my-account/

### Theme ###

Name: Tyche
Version: 1.0.11
Author URL: https://colorlib.com/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that
you did not build personally we recommend using a child theme. See: How to
create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: tyche/woocommerce/archive-product.php
tyche/woocommerce/global/sidebar.php
tyche/woocommerce/loop/add-to-cart.php
tyche/woocommerce/single-product/add-to-cart/external.php
tyche/woocommerce/single-product/add-to-cart/grouped.php
tyche/woocommerce/single-product/add-to-cart/simple.php
tyche/woocommerce/single-product/add-to-cart/variation-add-to-cart-button.php
tyche/woocommerce/single-product/tabs/tabs.php
tyche/woocommerce/single-product.php


### PBC Settings ###

Base location: TH:TH-50
Price Based On: billing
Shipping: –
Test mode: ✔
Test country: TH
`
Cheers,
--
Koh Zhiwei | Creative
M: +65 9693 1845 | T: +65 6557 2825
E: zhiwei.koh@onyxisland.com

Onyx Island Pte. Ltd.
2 Kallang Avenue #09-26
CT Hub@Kallang
Singapore 339407
http://www.onyxisland.com

**Comment by mikejolley** [comment]
As stated on the new issue form, issues should only be logged here once you've confirmed core is at fault, by disabling other plugins and switching to default themes. 

```
## Isolating the problem

- [ ] This bug happens with only WooCommerce plugin active
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/)
- [ ] I can reproduce this bug consistently using the steps above
```

Since this has not been done, this issue has not had other reports, and our clean installs do not have the problem, we can only assume it's a plugin conflict. I suggest you rule those things out first.

Thanks

**Comment by amiut** [comment]
![Screen shot](https://pasteboard.co/HSAcvCP.png)

Just had the same issue, As you can see i have around 1500 orders but it's showing nothing, 
I Deactivated all plugins, switched back to storefront theme , re-installed wordpress, Updated woocommerce database ( from Woocommerce > status > tools ) , still not working 
However if i click on Processing or completed orders links , it will show list of orders but not working for `all` and `mine`

**Comment by rrennick** [comment]
@amiut see #22271

**Comment by evadroid** [comment]
Just having exact the same issue. It happened after wordpress 5.0.2 update i guess.
